an introductory guide to WAI, why it exists and what it does (#1118)
a11yproject/a11yproject.com: src/posts/2020-10-02-introduction-to-WAI.md

---
layout: post
title: Introduction to Web Accessibility Initiative.
description: Key facts about web accessibility initiative, what it is, and why it exists.
category: Backgrounds
author: Hamsa Harcourt
further_reading:
  - url: https://www.w3.org/WAI/about/ 
    title:  About W3C WAI
    source: W3C Web Accessibility Initiative
  - url:  https://www.washington.edu/doit/what-web-accessibility-initiative-wai
    title: "What is Web Accessibility Initiative (WAI)?"
    source: "DO-IT - Disabilities, Opportunities, Internetworking, and Technology"
  - url: https://en.wikipedia.org/wiki/Web_Accessibility_Initiative
    title: Web Accessibility Initiative
    source: Wikipedia
date: 2020-10-02
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - background
---

> The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/) was created.

Reviewer: ```suggestion
The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by real or imagined borders. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/) was created.
```

I love that sentence, but am suggesting removing the semicolon to help keep the reading level lower.

> The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/) was created.

Reviewer: ```suggestion
The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (<abbr>WAI</abbr>)](https://www.w3.org/WAI/) was created.
```

This will help codify this acronym for later uses. [Here's a little more info](https://developer.paciellogroup.com/blog/2019/03/short-note-the-abbreviation-appreciation-society/), if you're interested.

## Introduction

WAI is one of the initiatives created by the [World Wide Web Consortium(W3C)](https://www.w3.org/Consortium/) as an effort to improve accessibility on the web.

According to Tim Bernes-Lee, Director of the W3C and inventor of the World Wide Web:

<blockquote>
	<p>Worldwide, there are more than 750 million people with disabilities. As we move towards a highly connected world, it is critical that the web be usable by anyone, regardless of individual capabilities and disabilities.</p>
</blockquote>

WAI pursues accessibility through the following ways:

- Ensuring that web technologies support accessibility.
- Developing guidelines for web content, applications, browsers, and authoring tools.
- Developing tools to evaluate and facilitate accessibility.
- Supporting education and outreach on web accessibility.
- Co-ordinating with research and development that may impact the future of accessibility on the web.
- Promoting harmonized international uptake of web accessibility standards.

## Organization

> WAI consists of several [working groups](https://en.wikipedia.org/wiki/Working_group) and [special interest groups](https://en.wikipedia.org/wiki/Special_Interest_Group). Each working together for a common goal: improve accessibility on the web. The following are the working and special interest groups :

Reviewer: ```suggestion
WAI consists of several [working groups](https://en.wikipedia.org/wiki/Working_group) and [special interest groups](https://en.wikipedia.org/wiki/Special_Interest_Group). Each group works on a specific subject, but they are all in the pursuit of a common goal: improving accessibility on the web. The following are the working and special interest groups :
```

Added a bit of clarity here about why there are separate groups.

- [Accessibility Guideline Working Group (formerly the Web Content Accessibility Guidelines Working Group)](https://www.w3.org/WAI/GL/).
- [Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/).
- [Accessible Rich Internet Applications (ARIA) Working Group](https://www.w3.org/WAI/ARIA/).
- [Education and Outreach Working Group (EOWG)](https://www.w3.org/WAI/about/groups/eowg/).
- [WAI Interest Group (WAI IG)](https://www.w3.org/WAI/about/groups/waiig/).

## Summary

> WAI was created by the [W3C](https://www.w3.org/Consortium/) as an effort to improve accessibility on the web. They develop accessibility standards and offer implementation guidance. WAI is made up of several working groups and special interest groups working together for a common goal: improve accessibility.

Reviewer: ```suggestion
WAI was created by the [W3C](https://www.w3.org/Consortium/) as an effort to improve accessibility on the web. They develop accessibility standards and offer implementation guidance. WAI is made up of several working groups and special interest groups working together for a common goal: improving accessibility.
```

> WAI was created by the [W3C](https://www.w3.org/Consortium/) as an effort to improve accessibility on the web. They develop accessibility standards and offer implementation guidance. WAI is made up of several working groups and special interest groups working together for a common goal: improve accessibility.

Reviewer: >improving accessibility.

Improving accessibility for who?
